### DISCUSSION TITLE
migrated database from mysql to mariadb [Sep 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3918] [Discord Threads]
> 🙋‍♂️ **tosmatt**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### at the moment, there is no easy way to handle this.. as the credentials are kept in encrypted way within nocodb..    
####    
#### https://github.com/nocodb/nocodb/issues/3700  
#  
> 💁 **o1lab**  
#### have added above issue  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1021316249898188802)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
